File: Swift/프로그래머스/0/120907. OX퀴즈/OX퀴즈.swift
import Foundation

func solution(_ quiz:[String]) -> [String] {
    var answer = [String]()
    
    for operation in quiz {
        let operationArray = operation.split(separator: " ")
        
        let num1 = Int(operationArray[0]) ?? 0
        let operation = String(operationArray[1])
        let num2 = Int(operationArray[2]) ?? 0
        let NumAnswer = Int(String(operationArray[4])) ?? 0
        
        var guess = 0
        if operation == "+" {
            guess = num1 + num2
        } else {
            guess = num1 - num2
        }
        
        if guess == NumAnswer {
            answer.append("O")
        } else {
            answer.append("X")
        }
    }
    return answer
}

File: Python3/프로그래머스/0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    answer = []
    
    for operation in quiz:
        operation_array = operation.split(" ")
        
        num1, operation, num2, num_answer = int(operation_array[0]), operation_array[1], int(operation_array[2]), int(operation_array[4])
        
        guess = 0
        
        if operation == "+":
            guess = num1 + num2
        else:
            guess = num1 - num2
        
        if guess == num_answer:
            answer.append("O")
        else:
            answer.append("X")
            
    return answer